cumulus/parachains/integration-tests/emulated/tests/people/people-rococo/Cargo.toml

```
@@ -0,0 +1,37 @@
+[package]
+name = "people-rococo-integration-tests"
+version = "1.0.0"
```

```suggestion
version = "0.1.0"
```

Maybe a nit, but I wouldn't say this is a major.